io.designer.GravitDesigner.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: io.designer.GravitDesigner
+base: io.atom.electron.BaseApp
+base-version: '18.08'
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: gravit-designer
+tags:
+  - proprietary
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=home
+modules:
+  - name: gravit-designer
+    buildsystem: simple
+    build-commands:
+      - install -Dt /app/bin apply_extra gravit-designer
+      - install -Dm644 -t /app/share/metainfo io.designer.GravitDesigner.appdata.xml
+      - install -Dm644 -t /app/share/applications io.designer.GravitDesigner.desktop
+      - install -Dm644 -t /app/share/icons/hicolor/64x64/apps io.designer.GravitDesigner.png
```

Comment:
If they have a 128px icon available as well it'd look slightly nicer in app stores before install. Not compulsory though.